src/components/task-plan/homework/exercises.cjsx

```diff
@@ -183,6 +183,11 @@ ExerciseTable = React.createClass
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
     content = ExerciseStore.getContent(exerciseId)
+    contentDiv = document.createElement('div')
+    contentDiv.innerHTML = content
+    
+    if contentDiv.children.length > 1
```

Comment:
_Minor:_ hmm, sorry, looking at this a bit more closely. Could the same thing be accomplished by doing `.td > :not(::first-child) { display: none; }`? If so, maybe that would be quicker than creating an element, pulling out the 1st child, converting it back to a string, then having react convert it back to DOM nodes?

/cc @pandafulmanda @darrenmason @nathanstitt ; Thoughts on trade-offs?